lib/parse.js

```diff
@@ -2236,17 +2236,86 @@ function parse($TEXT, options) {
         })
     }
 
+    function import_nameAsterisk() {
+        var start = S.token;
+        var foreign_name;
+        var name;
+
+        next();
+
+        var end = prev();
+
+        name = new AST_Symbol({
```

Comment:
I'm no expert on ES6, let alone the parser. So I think I'll defer to @avdg for this case.